apps/dashboard/app/assets/stylesheets/application.scss

```diff
@@ -47,26 +56,27 @@ $ood-blue: rgb(0, 95, 133);
 
 
 // Load up bootstrap/font-awesome style sheet
-@import "bootstrap";
 @import "font-awesome";
 
-@import "ood_appkit";
-
 // Load up dataTables bootstrap specific styles
-@import "dataTables/bootstrap/3/jquery.dataTables.bootstrap";
+@import "../node_modules/datatables.net-bs4/css/dataTables.bootstrap4";
```

Comment:
Since node_modules is still in the path, and we can deal with that for now (though need to definitely test performance due to this), it should work to do

    @import "datatables.net-bs4/css/dataTables.bootstrap4";